data-plane/dispatcher/src/main/java/dev/knative/eventing/kafka/broker/dispatcher/Main.java

```diff
@@ -141,21 +144,26 @@ public static void main(final String[] args) throws IOException {
           logger.error("Consumer deployer not started", t);
           vertx.close(v -> System.exit(1));
         });
-      waitConsumerDeployer.await(5, TimeUnit.SECONDS);
+      if (!waitConsumerDeployer.await(env.getWaitStartupSeconds(), TimeUnit.SECONDS)) {
+        throw new TimeoutException("Failed to deploy consumer deployer");
+      }
```

Comment:
This `waitConsumerDeployer.await` does nothing because it returns a boolean based on the CounteDownLatch value after the timeout, so we need to check the return value.